library/src/main/scala/scala/scalajs/js/Object.scala

```diff
@@ -202,6 +220,38 @@ object Object extends Object {
    */
   def preventExtensions(o: Object): o.type = native
 
+  /** <span class="badge badge-ecma2015" style="float: right;">ECMAScript 2015</span>
+   *
+   *  Object.is() determines whether two values are the same value. Two values
+   *  are the same if one of the following holds:
+   *
+   *  <ul>
+   *    <li>both undefined
+   *    <li>both null
+   *    <li>both true or both false
+   *    <li>both strings of the same length with the same characters in the same
+   *        order
+   *    <li>both the same object (means both object have same reference)
+   *    <li>both numbers and
+   *      <ul>
+   *        <li>both +0
+   *        <li>both -0
+   *        <li>both NaN
+   *        <li>or both non-zero and both not NaN and both have the same value
+   *      </ul>
+   *    </li>
+   *  </ul>
+   *
+   *  This is not the same as being equal according to JavaScript's `===`
+   *  operator (exposed as
+   *  [[scala.scalajs.js.special.strictEquals js.special.strictEquals]] in
+   *  Scala.js). The `===` operator treats the number values `-0` and `+0` as
+   *  equal and treats `NaN` as not equal to `NaN`.
+   *
+   *  MDN
+   */
+  def is(value1: Any, value2: Any): Boolean = native
```

Comment:
The arguments should still be `scala.Any`, here, though.

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/3869/commits/61834108bab2e28f4d3526a98b4d6da073c4b820